external_ssl_proxy_lb_mig_backend_custom_header/main.tf

```diff
@@ -30,7 +30,6 @@ resource "tls_private_key" "default" {
 }
 
 resource "tls_self_signed_cert" "default" {
-  key_algorithm   = tls_private_key.default.algorithm
```

Comment:
Why remove?

Reply:
This is now a readonly field in the latest version
https://github.com/hashicorp/terraform-provider-tls/blob/main/CHANGELOG.md#400-july-21-2022
https://registry.terraform.io/providers/hashicorp/tls/latest/docs/resources/self_signed_cert#key_algorithm